### ISSUE TITLE
Clarify difference between "ticket" and "badge"

### ISSUE THREAD

**Issue by pferreir** [indico/indico#5092]
This is a common source of confusion for users. It's not clear what the difference is. Our current interpretation seems to be:
* Tickets - they have a "ticket" template on either side, or at least one "ticket" field (i.e. a QRCode)
* Badges - they can contain anything which is not a "ticket" field or template

I'm not sure it makes even much sense to differentiate the two so much at the UI level - the print dialogs are identical. This distinction was introduced mostly to deal with CERN's access workflow use case, where the ability to print a ticket may depend on whether personal data has been provided. I wonder if we cannot just say it's all badges, which it actually is under the hood...

**Comment by bpedersen2** [comment]
Yes, these are a bit confusing. One  way to differentiate would be:

ticket: is issued online before the event and the user is expected to bring it with him( either as printout or digital?)
badge: is prepared by the event team and issued to the participants during check-in.

As for the ticket,besides the templated pdf a pure digital format compatible with wallet apps would be a  nice extension (e.g .pkpass, see https://developer.apple.com/documentation/walletpasses  which are zip files with a special content).

**Comment by ThiefMaster** [comment]
Passbook files are a great idea in principle, but the fact that you can't just generate one but need to get a certificate from Apple to create/sign one is somewhat ridiculous... :/
